### PR TITLE
Fix #34 Add an npm script to build the xpi.

### LINKED ISSUE #34: Add npm script to build XPI?
```diff
diff --git a/package.json b/package.json
index fed776a..9b2241b 100644
--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
   },
   "scripts": {
     "addon": "cd addon && mkdir -p ../Profile/ && jpm run -b nightly --prefs ../.jpm-prefs.json --profile ../Profile/ --no-copy --binary-args -jsconsole",
+    "build": "jpm xpi --addon-dir addon --dest-dir=. && mv null.xpi $npm_package_name-$npm_package_version.xpi",
     "lint:css": "stylelint 'addon/**/*.css'"
   }
 }
```

Running `npm run build` will now generate a file in the root directory named "tab-split-0.1.0.xpi" (so we'd want to add `*.xpi` to the .gitignore file probably.

Not sure why the file is being `null.xpi`. Maybe we need an "addon/manifest.json" file with values copied from package.json. Currently running `npx addons-linter tab-split-0.1.0.xpi` returns the following:

```sh
$ npx addons-linter tab-split-0.1.0.xpi

Validation Summary:

errors          0
notices         1
warnings        0

NOTICES:

Code                    Message         Description                     File   Line   Column
TYPE_NO_MANIFEST_JSON   manifest.json   No manifest.json was found at
                        was not found   the root of the extension.
                                        The package file must be a
                                        ZIP of the extension's files
                                        themselves, not of the
                                        containing directory. See:
                                        https://mzl.la/2r2McKv for
                                        more on packaging.

$ echo $? # 0
```

### PR DESCRIPTION
**Pull request by fzzzy**
Also add scripts to run everything and just run firefox with the addon
installed, but run is not fully functional until bug https://bugzil.la/1434076 is fixed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "url": "git+https://github.com/mozilla/tab-split.git"
   },
   "scripts": {
+    "start": "npm-run-all lint build run",
```
Comment (pdehaan) [inline package.json:25]: Not sure if anybody is rocking Windows (or if Win 10.x supports a real Unixy language yet w/ PowerShell, Bash, etc), or if we need to fall back to something like npm [**mkdirp**](https://www.npmjs.com/package/mkdirp) for cross OS greatness.

File: package.json [hunk 2]
```diff
@@ -22,6 +22,9 @@
...
+    "build": "mkdir -p ./Profile && web-ext build --source-dir=webext",
+    "run": "mkdir -p ./Profile && echo 'Starting the tab split extension automatically from the command line will fail until bug 1434076 is fixed. In the meantime please use about:debugging to load both experiment/schema.json and webext/manifest.json' && echo 'Some day we can run this here: web-ext run --source-dir=webext -p ../Profile'",
```
Comment (pdehaan) [inline package.json]: Should probably just be "./Profile" at the end of the second `echo` statement. ¯\\\_(ツ)\_/¯

File: package.json (hunk 1 above)
Comment (fzzzy) [reply package.json:25]: Supporting windows sounds good to me. I don't use it so can't test it.

Comment (pdehaan) [reply package.json:25]: The other alternative is just check in an empty ./Profile directory with a nested .gitignore file which ignores everything. That way we aren't constantly trying to constantly re-run `mkdir -p ./Profile` (which would get called twice on `npm start`).

Comment (pdehaan) [reply package.json:25]: But i'm also happy to just merge this as-is and we can over optimize later when the Bugzilla bug is fixed.

Comment (fzzzy) [reply package.json:25]: Let's just add a bug about supporting windows.

Comment (fzzzy) [reply package.json:25]: https://github.com/mozilla/tab-split/issues/54

**Pull request merged**